bot/processors/pennychat.py

```diff
@@ -142,6 +150,48 @@ def penny_chat_details_modal(penny_chat_invitation):
                     }
                 ]
             },
+            {
+                "type": "divider"
+            },
+            {
+                'block_id': 'penny_chat_visibility',
+                'type': 'section',
+                'text': {
+                    'type': 'mrkdwn',
+                    'text': '*Chat Visibility*',
+                },
+                'accessory': {
+                    'action_id': 'penny_chat_visibility',
+                    'type': 'static_select',
+                    'placeholder': {
+                        'type': 'plain_text',
+                        'text': 'Select visibility'
+                    },
+                    'options': [
+                        {
+                            'text': {
+                                'type': 'plain_text',
+                                'text': 'Private Chat',
+                            },
+                            'value': '20',
```

Comment:
Can these be ints instead of strings?

Reply:
For some reason (I believe it's just the way Slack API processes the JSON), using ints causes a syntax error and you can't dispatch the modal.